config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file about?

Reply:
This is where we can specify the cookie serializer for serializing the cookies.